traffic_ops/traffic_ops_golang/deliveryservice/request/requests.go

```diff
@@ -92,6 +92,7 @@ func (req *TODeliveryServiceRequest) Read(h http.Header, useIMS bool) ([]interfa
 		"author":     dbhelpers.WhereColumnInfo{Column: "a.username"},
 		"authorId":   dbhelpers.WhereColumnInfo{Column: "r.author_id", Checker: api.IsInt},
 		"changeType": dbhelpers.WhereColumnInfo{Column: "r.change_type"},
+		"createdAt":  dbhelpers.WhereColumnInfo{Column: "r.created_at"},
```

Comment:
if you're adding a query parameter, each affected version will need a documentation update.

Reply:
it's not a new query param. it's a new value to order by. appears those are not documented currently.

Reply:
or at least i thought it was. now i'm questioning myself...

Reply:
I'm pretty sure this both adds a new query parameter and a new orderBy option.

Reply:
yeah, it's so orderby=createdAt will be respected. not sure anyone would want to ?createdAt=date so maybe we should leave it undocumented? or should I document it? your call.

Reply:
It might be a niche use-case, but IMO it's weirder to be able to filter by all properties except one.

The really tough question for me is one you don't need to answer here: when olderThan and newerThan are implemented for this endpoint, will the use createdAt - which makes the most sense - or lastUpdated - which is what every other endpoint will use.

Reply:
not sure what the date format needs to be. this doesn't seem to return anything when i know there is one with this created date: ?createdAt=2020-10-29T22:02:42Z. will dig...

Reply:
oh wait. milliseconds. yeah, this isn't going to be very valuable...

Reply:
this works GET api/3.0/deliveryservice_requests?createdAt=2019-09-19T19:35:38.828535Z which good luck figuring that out...

Reply:
what is the name of this date format? do you know? i'm guessing you do. 2019-09-19T19:35:38.828535Z

Reply:
RFC3339

Reply:
updated